zero_to_chat_with_your_data/zero_to_chat_with_your_data.md

```diff
@@ -892,10 +917,83 @@ The Streamlit in Snowflake interface is divided into three panes:
 
 By default, only the Streamlit editor and preview panes are displayed. To change the display, use the show/hide buttons in the lower-left corner of the Streamlit in Snowflake editor.
 
-> 🚧 UNDER CONSTRUCTION: Provide a screenshot of the default Streamlit application
+![streamlit](assets/streamlit_2.png)
 
 Add snowpark-ml-python package from the packages dropdown in the code editor section
 
+### Cortex AI
+
+Snowflake offers two broad categories of powerful, intelligent features based on Artificial Intelligence (AI) and Machine Learning (ML). These features can help you do more with your data in less time than ever before.
+
+- Snowflake Cortex is a suite of AI features that use large language models (LLMs) to understand unstructured data, answer freeform questions, and provide intelligent assistance. This suite of Snowflake AI Features comprises:
+
+  - Snowflake Cortex LLM Functions
+
+  - Universal Search
```

Comment:
Can you also provide a one-sentence summary of each of these features?

Reply:
Added